modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBAdminNavigationDisplayContext.java

```diff
@@ -129,6 +132,29 @@ public JSONArray getKBFolderDataJSONArray() throws PortalException {
 			));
 	}
 
+	public long getKBObjectToMoveParent() throws PortalException {
+		String kbObjectToMoveType = ParamUtil.getString(
+			_httpServletRequest, "kbObjectToMoveType");
+
+		long kbObjectToMoveId = ParamUtil.getLong(
+			_httpServletRequest, "kbObjectToMoveId");
+
+		if (kbObjectToMoveType.equals(KBConstants.TYPE_FOLDER)) {
+			KBFolder kbFolder = KBFolderLocalServiceUtil.getKBFolder(
+				kbObjectToMoveId);
+
+			return kbFolder.getParentKBFolderId();
+		}
+
+		int articleVersion = ParamUtil.getInteger(
+			_httpServletRequest, "kbObjectVersion", -1);
```

Comment:
For consistency, shouldn't this be `kbObjectVersion`? In general, parameter names and variable names should be the same.